dataminer/Functions/Video_Thumbnails/Configuring_Video_Thumbnails/Adding_parameters_to_the_video_URL.md

```diff
@@ -0,0 +1,112 @@
+
+- `user=` and `password=` (Images): You can add a password and username in the URL for basic HTTP authentication. However, in that case we advise to always use HTTPS, as otherwise the username and password will not be encrypted. Note that this parameter can only be used when the `proxy=` parameter is set to `true` (default).
+
+  Example: `#https://<DMA>/VideoThumbnails/Video.htm?type=Generic%20Images&source=http%3A%2F%2F10.0.20.101%2Fimages%2Fthumbs%2F4.jpg&user=admin&password=test&refresh=1000`
+
+- `auth=` (Images): Specify an HTTP authorization header that will be added to the HTTP request when a thumbnail image is requested from the video server. This option is required when the video server expects an authentication token (for example *OAuth2*). Note that this parameter can only be used when the `proxy=` parameter is set to `true` (default).
+
+  Example: `#https://<DMA>/videothumbnails/video.htm?type=Generic%20Images&source=https%3A%2F%2F77.158.55.113%2Fvos-api%2Fmonitor%2Fv1%2Fservices%2F55002da8-37fd-43de-82a9-f6b75089d8c9%2Fthumbnail&auth=bearer%20580a4efa-0aab-4882-af91-7b0118c67f5d`
+
+- `volume=` (All video types): Available from DataMiner 10.2.0 [CU1]/10.2.4 onwards. Specify the volume for the video player in the URL. The volume should be specified as a percentage, ranging from 0 (i.e. muted) to 100.
+
+  Example: `#https://<DMA>/VideoThumbnails/Video.htm?type=VLC&source=https://videoserver/video.mp4&volume=50`
+
+- `loop=` (All video types): Available from DataMiner 10.2.0 [CU1]/10.2.4 onwards. Specify that the video should play continuously in a loop. Set to `false` by default.
+
+  `#https://<DMA>/VideoThumbnails/Video.htm?type=HTML5&source=https://videoserver/video.mp4&loop=true`
+
+- `fitMode=` (Images): Available from DataMiner 10.4.0 [CU18]/10.5.0 [CU6]/10.5.9 onwards<!--RN 43388-->. Specify how the image in a video thumbnail should be displayed.
+
+  The possible values are:
+
+  - `fill`: The image will completely cover the container. It may crop parts of the image, but it ensures that there is no empty space (*default*).
+
+  - `fit`: The image will be fully visible inside the container while maintaining aspect ratio. There may be empty space if aspect ratios differ.
+
+  - `stretch`: The image will stretch to exactly fill the container, ignoring aspect ratio. This may cause distortion.
+
+  - `center`: The image will retain its original size and will be aligned at the center. It may overflow or be cropped.
+
+  - `shrink`: The image will behave like `fill` or `center`, whichever results in a smaller image. It will only scale down if needed.
+
+  Example: `#http://<DMA>/VideoThumbnails/Video.htm?type=Generic%20Images&source=http://IpAddress/Folder/Picture.png&refresh=5000&fitMode=center`
+
+- `toolbar=false`/`showtoolbar=false` (All video types): By default the VLC toolbar is shown. To hide the toolbar, specify "toolbar=false" or "showtoolbar=false" in the URL.
```

Comment:
I would remove "VLC" here. For type `VLC` it's indeed the VLC toolbar, but for type `HTML5` it is the HTML5 toolbar.